src/app.md

```diff
@@ -271,6 +272,54 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-file-skip-dot] #ex-file-skip-dot
+<a name="ex-file-skip-dot"></a>
+##  Traverse directories while skipping dotfiles
+
+Iterate over the entries that are not hidden files or hidden directories. 
```

Comment:
How about:

Iterates over filesystem entries efficiently skipping hidden files and directories with help of [`WalkDir::filter_entry`].